CONTRIBUTING.md

```diff
@@ -0,0 +1,284 @@
+# Contributing Guide
+
+## Welcome to Percona Server for MySQL!
+
+We're excited to have you join the Percona community and participate in keeping open-source open.
+
+### Why Your Contribution Matters?
+
+Main driving force of open-source project is community. We, as developers team, highly appreciate any contributions.
+
+It may be advisable to initially submit bug fixes to the upstream project that will eventually become part of the next
+release of Percona. Alternatively, Percona provides expedited response times, transparent development processes, and
+streamlined contribution acceptance within a public repository.  
```

Comment:
Working with the MySQL 8.0 source code can be time-consuming, especially for developers new to the project.

MySQL is a complex piece of software with a vast codebase. Understanding the intricacies of its architecture, data structures, and algorithms can require significant time and effort. The codebase is constantly evolving, so staying up-to-date with changes can be challenging.

Here are some factors that contribute to the complexity of working with MySQL source code:

Large codebase: MySQL has a massive codebase, making it difficult to navigate and understand.
Complex architecture: MySQL architecture is intricate and involves multiple components and layers.
Performance optimizations: MySQL is heavily optimized for performance, which can make the codebase even more complex.
Ongoing development: The MySQL project constantly evolves, with new features and bug fixes regularly added.

However, for experienced C++ developers who are familiar with database systems and are willing to invest the time and effort, working with the MySQL source code can be a rewarding experience. It can provide a deep understanding of how MySQL works and allow for customization and optimization.

Reply:
text added